src/OpenLoco/src/Vehicles/OrderManager.cpp

```diff
@@ -42,7 +59,7 @@ namespace OpenLoco::Vehicles::OrderManager
         const auto offset = head->orderTableOffset;
         const auto size = head->sizeOfOrderTable;
 
-        sub_470795(offset, -size);
+        reoffsetVehicleOrderTables(offset, -size);
 
         // Fold orders table left to remove empty orders
         std::rotate(&orders()[offset], &orders()[offset + size], &orders()[numOrders()]);
```

Comment:
Isn't this just the same as shift orders down?

Reply:
No, this is working on the vehicle structs rather than the orders.

Reply:
Oh, you're right! That's a concise way of programming it. This function is clearing out an entire order table, though, so I suggest we refactor just this particular line out to `shiftOrdersDown`.